cli/commands/remote/workspace/delete/index.md

```diff
@@ -15,10 +15,10 @@ kbc remote workspace delete [flags]
 ### Options
 
 `-W, --workspace-id string`
-: Id of the workspace to delete. You can find it using the [List Workspaces](/cli/commands/remote/workspace/list/) command.
+: Id of the workspace to be deleted. You can find it using the [List Workspaces](/cli/commands/remote/workspace/list/) command.
```

Comment:
@jprochazk na to by si mal zalozit task (ked tu docs mergujes) a upravit wording aj v CLI samotnom.